tests/unit_test/data_processor/data_processor_test.py

```
@@ -1524,20 +1949,20 @@
+    def test_validate_data_sync_type_does_not_exist(self):
         bot = 'test_bot'
         collection_name = 'groceries'
         primary_key_col = "id"
         data = [{"id": 1, "item": "Juice", "price": 2.50, "quantity": 10}]
-        event_type = 'non_existent_event_type'
+        sync_type = 'non_existent_sync_type'
 
         processor = CognitionDataProcessor()
 
-        with pytest.raises(AppException, match=f"Event type does not exist"):
+        with pytest.raises(AppException, match=f"Sync type does not exist"):
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)